### PR TITLE
Handle login similar to PlonePAS

### LINKED ISSUE #7: Fire UserLoggedInEvent on login
Events:
- Products.PlonePAS.events.UserLoggedInEvent
- Products.PlonePAS.events.UserInitialLoginInEvent

### PR DESCRIPTION
**Pull request by phgross**
Fire login events, expire clipboard, create member area and set login times.

Closes #7.

@buchi please have a look ...

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -4,7 +4,9 @@ Changelog
 1.0.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Handle login similar that PlonePas(fire login events,
```
Comment (buchi) [inline docs/HISTORY.txt]: ...similar to PlonePAS (fire...

File: ftw/casauth/plugin.py [hunk 2]
```diff
@@ -104,13 +110,63 @@ def authenticateCredentials(self, credentials):
...
+        self.expire_clipboard()
+
+        mtool.createMemberArea(member_id=userid)
+
         pas.updateCredentials(self.REQUEST, self.REQUEST.RESPONSE, userid, '')
+        return member
 
-        return userid, userid
+    def set_login_times(self, member):
+        """The return value indicates if this is the first logged
```
Comment (buchi) [inline ftw/casauth/plugin.py]: A docstring makes this method publishable through the web. Make a comment instead.

**Review by buchi — CHANGES_REQUESTED** [review]
looks ok apart from the two things mentioned.

**Comment by phgross** [comment]
@buchi updated

File: docs/HISTORY.txt [hunk 3]
```diff
@@ -4,6 +4,10 @@ Changelog
 1.0.1 (unreleased)
 ------------------
 
+- Handle login similar to PlonePas(fire login events,
```
Comment (buchi) [inline docs/HISTORY.txt]: still two things: .PAS in capital letters (PlonePAS) and a space after PAS

**Comment by phgross** [comment]
@buchi ah sorry didn't realized that, updated again.

**Review by buchi — APPROVED** [review]

**Pull request merged**